ruby-training/trainees/david/test_katas/01/calculator.rb

```diff
@@ -0,0 +1,23 @@
+class Calculator
+
+    def self.add(a, b)
```

Comment:
Remember to write descriptive variable names (ex. "left_operand" and "right_operand")